plot_app/templates/main.js

```diff
@@ -7,6 +7,30 @@ $.fn.scrollView = function () {
     });
 }
 
+$(".chosen-select").chosen({
+    no_results_text: "Oops, nothing found!",
+    width: "100%"
+  });
+
+ $( "#error-label" )
+  .change(function () {
+    var error_labels = "";
+    $( "select option:selected" ).each(function() {
+        error_labels += $( this ).text() + ",";
+       //error_labels.push($( this ).text())
+    });
+
+
```

Comment:
Nitpick: can you remove this empty line, otherwise the ajax request looks like a separate thing outside the current block. Can you also use 4 spaces as indentation?